circle.yml

```diff
@@ -4,5 +4,5 @@ dependencies:
 
 test:
     override:
-        - tox -e py27,py30,py31,py34
+        - tox -r
```

Comment:
Nope, it's done. I forgot to [fix machine pyenv](https://circleci.com/docs/build-image-precise/#python) into the `circle.yml` file to allow all python versions we need for the tests.
It's green now. :wink: